src/build.jl

```diff
@@ -1,23 +1,23 @@
 ################
 # LaTeX-engine #
 ################
-@enum(LaTeXEngine, LUALATEX, PDFLATEX)
+@enum(LaTeXEngine, LUALATEX, PDFLATEX, XELATEX)
 
 """
 The active LaTeX engine. Initialized the first time [`latexengine`](@ref) is called.
 """
 const ACTIVE_LATEX_ENGINE = Ref{Union{Nothing, LaTeXEngine}}(nothing)
 function latexengine()
     if ACTIVE_LATEX_ENGINE[] === nothing
-        for (engine, enum) in zip(("lualatex", "pdflatex"), (LUALATEX, PDFLATEX))
+        for (engine, enum) in zip(("lualatex", "pdflatex", "xelatex"), (LUALATEX, PDFLATEX, XELATEX))
```

Comment:
Do we want to pick `xelatex` over `pdflatex` if we find it?

Reply:
I don't know much about XeLaTeX, so I would stick with the ordering above and leave it to the user to request something else.